teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+
+# Class representing a student
+class Teacher < Person
+  def initialize(name, age, specialization)
```

Comment:
- [ ] it would be better to add permission and should be true by default and pass it to the super  keyword in order to  match the requirements

Reply:
Thank you @youmari. Done!